jscomp/build_tests/super_errors/fixtures/private_without_body.res

```diff
@@ -0,0 +1,2 @@
+%%private
+let a = 2
```

Comment:
Any way to make the error message more approachable. Not so clear why `a` "does not have a body".

Reply:
The message highlights `%private`. It's not `a` that doesn't have a body, it's `%%private`.

![image](https://user-images.githubusercontent.com/16160544/232866910-6c0c3e93-a315-4974-b845-b44fc0ee6c14.png)


Maybe `%%private extension expects arguments`?

Reply:
Yes: private expects the definition as argument.

Reply:
>Error: The '%%private' extension expects a definition as its argument.

>Example usage:
>%%private
>let a = (x) => x + 2;

>In this case, the '%%private' extension is missing a definition as its argument. Please provide a valid definition for the private extension to work as intended.